app/decorators/merge_runners_request_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class MergeRunnersRequestDecorator < Draper::Decorator
+  delegate_all
+
+  def runners
+    object.runners.map { |r| h.link_to r.id, r }.join(', ').html_safe
```

Comment:
@simplay: see the readme of the draper gem for a general explanation of the Decorators, and an explanation of what ```object``` and ```h``` is: https://github.com/drapergem/draper#accessing-the-model,  https://github.com/drapergem/draper#accessing-helpers